grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -151,11 +169,20 @@ class GrabbitConfiguration {
     @CompileStatic
     static class PathConfiguration {
         String path
+        Collection<String> excludePaths
         Collection<String> workflowConfigIds
 
+        void setPath(@Nonnull String path) {
+            this.path = path!=null && path.endsWith("/") ? path.substring(0, path.length() - 1): path
+        }
+
+        Collection<String> getAbsolutePaths(@Nonnull Collection<String> excludePaths) {
+            return excludePaths.collect { "${this.path}/${it}" }
```

Comment:
There's no invariant that `excludePaths` is non-null, so this can give an NPE...